bin/index.js

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env node
```

Comment:
Run `Format Document` before committing to have better readability?

Reply:
This is required. It specifies the user's device  (person who downloaded the package) to run the code in a node environment

Reply:
Missunderstood, due to github Showing the comment in first line of code